### ISSUE TITLE
use `<clr-wizard />` in a non-modal context

### ISSUE THREAD

**Issue by royling** [vmware/clarity#2426]
**Select one ...** (check one with "x")

```
[ ] bug
[x] feature request
[ ] enhancement
```

### Expected behavior

Now, `<clr-wizard />` wraps a `<clr-modal />` underneath, which makes sense to most use cases. However, do you consider to support using it in a non-modal context? Specifically, I mean:
* providing a context option so the user can choose whether it's in a modal or not
* or, providing this component in a context-agnostic way so that it can be composited with a modal

### Actual behavior

In my case, I was trying to use this component in a vSphere client plugin project. As you might know, the plugin SDK provides necessary modal JS API. But the extension point is only the modal body (ie. embedding the plugin view into the iframe in the `modal-body` element). Then, the wizard component can not be used at all in this case.

### Reproduction of behavior

* Include a link to the reproduction scenario you created by forking one of the Clarity Plunker Templates:
  
  
  
  

### Environment details

* **Angular version:** 4.x.x

* **Clarity version:**

* **OS and version:**

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by jewway** [comment]
Hi @royling , you can refer [Inline Wizard](http://new-clarity-grid.surge.sh) , 
The document seems to have not been updated yet , but it's work.

**Comment by royling** [comment]
@jewway Thanks for the demo and sample code, the inline wizard is exactly what I wanted.

I guess it works by adding `class="clr-wizard--inline clr-wizard--no-shadow clr-wizard--no-title"` to the host tag.

I'll keep this issue open for tracking doc updat, or possibly if it's still considered a valid feature requirement. 😛

**Comment by mathisscott** [comment]
Hi @jewway and @royling 
The inline/static wizard will remain in our demos. It is something we did explicitly for @royling's use case. But that is not a use case we want to promote to all users of Clarity because most of them aren't going to be projecting wizards into iframes inside of existing modals.

The styles are there and will remain there. But we won't be adding them to our documentation because it is not a pattern we want to promote. This is something we did in order to help out VMware developers (and third party developers writing plugins for VMware applications).
